bus-mapping/src/exec_trace.rs

```diff
@@ -0,0 +1,213 @@
+pub(crate) mod exec_step;
+use crate::evm::EvmWord;
+use crate::operation::{container::OperationContainer, Operation};
+use core::ops::{Index, IndexMut};
+pub(crate) use exec_step::ExecutionStep;
+use pasta_curves::arithmetic::FieldExt;
+
+/// Doc
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub(crate) struct BlockConstants<F: FieldExt> {
+    hash: EvmWord, // Until we know how to deal with it
+    coinbase: F,
+    timestamp: F,
+    number: F,
+    difficulty: F,
+    gaslimit: F,
+    chain_id: F,
```

Comment:
`basefee` seems to belong here, but it's fine to add in next PR.
```suggestion
    chain_id: F,
    basefee: F,
```

Reply:
Added it too. Thanks!